.github/ISSUE_TEMPLATE/7_cherry_pick.yml

```diff
@@ -39,6 +50,13 @@ body:
         The current branches can be found under release-caniddate-branch.version for [beta](https://github.com/flutter/flutter/blob/beta/bin/internal/release-candidate-branch.version) and [stable](https://github.com/flutter/flutter/blob/stable/bin/internal/release-candidate-branch.version)
     validations:
       required: true
+  - type: textarea
+    id: changelog_description
+    attributes:
+      label: Changelog Description
+      description: Explain this CP in one line that is accessible to most Flutter developers
```

Comment:
Can we add a link to https://github.com/flutter/flutter/wiki/Hotfix-Documentation-Best-Practices for an example on how to create these?